docs/containerd-2.0.md

```diff
@@ -0,0 +1,105 @@
+# containerd 2.0
+
+Please try out the release binaries and report any issues at <https://github.com/containerd/containerd/issues/>.
+
+## What's new
```

Comment:
I think I'd order the "what's new" here a bit differently, maybe something like this set of criteria:

1. New features/functionality or major enhancements (transfer service, sbserver by default)
2. Defaults changes (NRI, new config format version 3)
3. Regular (non-major) enhancements (igzip, otel configuration, introspection plugin info, deprecation warnings)

My goal is to have the most important/exciting information first.  What do you think about that?

Reply:
Iterated on in https://github.com/containerd/containerd/pull/10696/commits/b6ce8f17f59c4993614d2b214482f20c4c6569a4